tests/requirements.txt

```diff
@@ -13,3 +13,5 @@ sacred
 neptune-client
 twine==1.13.0
 pillow<7.0.0
+--extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0
+nvidia-dali
```

Comment:
just thinking if want to add this dependence to specific CUDA version, is there a version on PyPI? @williamFalcon